src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -629,11 +629,10 @@ export default {
 		}
 	},
 	mounted() {
-		const urlHash = this.$router.currentRoute?.value?.hash;
-
-		if (urlHash) {
-			const elementToScrollTo = document.querySelector(urlHash);
-			const topOfSectionToScrollTo = (elementToScrollTo?.offsetTop ?? 0) - 10 ?? 0;
+		const sectionId = this.$route?.query?.goTo || '';
```

Comment:
I don't think we can replace the hash completely, since we might be using the URLs somewhere. Can we support hash or `goTo`?